README.md

```diff
@@ -36,7 +36,6 @@ This is the repository for [Kubernetes DNS](http://kubernetes.io/docs/admin/dns/
 ## Release process
 
 1. Build and test (`make build` and `make test`)
-1. Update [go dependencies](docs/go-dependencies.md) if needed.
```

Comment:
Hoping dep could be good enough so that we won't need a substitute of the godep script we have...